client_plugin/drivers/shared/etcdops.go

```diff
@@ -346,14 +520,14 @@ func (d *VolumeDriver) etcdList() ([]string, error) {
 }
 
 // writeVolMetadata - Update or Create volume metadata in KV store
```

Comment:
Nit: "writeVolMetadata" -> "WriteVolMetadata" to be consistent with the function name.